frontend/src/pages/chat/components/chat-date/styles.module.scss

```
@@ -0,0 +1,28 @@
+.date {
+  display: flex;
+  color: var(--blue-200);
+  font-weight: 500;
+  font-size: 12px;
+  white-space: nowrap;
+}
+
+.date::before {
+  content: "";
+  display: inline-block;
+  align-items: center;
+  width: 100%;
+  height: 1px;
+  margin: auto 0;
+  margin-right: 45px;
+  background-color: var(--blue-100);
+}
+
+.date::after {
+  content: "";
+  display: inline-block;
+  width: 100%;
+  height: 1px;
+  margin: auto 0;
+  margin-left: 45px;
+  background-color: var(--blue-100);
+}
```

maybe it is not necessary right now, but we can combine the same styles from pseudo-elements and define separately only styles that are different

```suggestion
.date::before,
.date::after {
  content: "";
  display: inline-block;
  width: 100%;
  height: 1px;
  margin: auto 0;
  background-color: var(--blue-100);
}

.date::before{
  align-items: center;
  margin-right: 45px;
}

.date::after {
  margin-left: 45px;
}
```

---

Done